docs/docs-content/clusters/edge/edgeforge-workflow/palette-canvos/build-content-bundle.md

```diff
@@ -0,0 +1,318 @@
+   at the top right side corner of the landing page below the **User drop-down Menu**.
+
+6. Navigate to the left **Main Menu** and select **Profiles**.
+
+7. Use the **Cloud Types drop-down Menu** and select **Edge Native**.
+
+8. Click on the cluster profile you want to include in the content bundle.
+
+9. You can find the cluster profile ID by reviewing the URL of the current page. The cluster profile ID is the last
+   value in the URL. Repeat this step for all the cluster profiles you want to specify in the content bundle.
+
+   ```text
+   https://console.spectrocloud.com/projects/yourProjectId/profiles/cluster/<YourClusterProfileHere>
+   ```
+
+10. (Optional) If your cluster profile uses images or helm charts that are hosted on private registries that require
+    authentication, you must provide a JSON file that contains the necessary credentials to access the registry.
+
+    <Tabs>
+
+    <TabItem value="helm" label="Helm">
+
+    For authenticated access to Helm charts, your must provide credentials with the following schema. Use a key at the
+    root level of the JSON object named "helm" and set its value to a list. The list is a list of credentials for each
+    Helm chart repository. For each set of credentials, use an object in the list with the keys "endpoint", "username",
+    and "password".
+
+    ```json
+    {
+      "helm": [
+          {
+            "endpoint": <Registry URL>,
+            "username": <Registry username>,
+            "password": <Password>
+          }
+      ]
+    }
+    ```
+
+    For example, the following JSON code is a valid set of credentials.
+
+    ```json
+    {
+      "helm": [
+        {
+          "endpoint": "harbor.abcd.com",
+          "username": "admin",
+          "password": "xxxxxxxx"
+        }
+      ]
+    }
+    ```
+
+    </TabItem>
+
+    <TabItem value="image" label="Image">
+
+    For image registries, you must provide credentials with the following schema. Provide a key at the root level of the
+    JSON object named "image" and set its value to a list. The list is a list of credentials for each Helm chart
+    repository. For each set of credentials, use an object in the list with the keys "endpoint", "username", and
+    "password".
+
+    ```json
+    {
+      "image": [
+          {
+            "endpoint": <Registry URL>,
+            "username": <Registry username>,
+            "password": <Password>
+          }
+      ]
+    }
+    ```
+
+    For example, the following JSON code provides access to two registries `ttl.sh` and `docker.io` with two
+    username-password pairs.
+
+    ```json
+    {
+      "image": [
+        {
+          "endpoint": "ttl.sh",
+          "username": "admin",
+          "password": "Welc0me!123"
+        },
+        {
+          "endpoint": "docker.io",
+          "username": "akhileshpvt",
+          "password": "Lucent122333!"
```

Comment:
```suggestion
          "password": "*********"
```
Let's not use mock passwords, gitleaks will complain and as a result we will have to add two commits to ignore file. One for this merge, and another after we merge to master. It's ugly 😢